hail/python/hail/experimental/plots.py

```diff
@@ -264,3 +269,148 @@ def scale_file_sizes(file_sizes):
         return Tabs(tabs=[Panel(child=entries_grid, title='Entries'), Panel(child=rows_grid, title='Rows')])
     else:
         return rows_grid
+
+def cnv_variant_cluster(mt, variant_range, show_plot=False, consecutive_interval=True):
+    """Create a (set of) scatterplot(s) showing SNP clusters per variant specified.
+
+    Parameters
+    ----------
+    mt : hail matrix table with locus, GTA, NORMX, NORMY, THETA values
```

Comment:
I think you should have a flag for cluster_type which can be one of `polar`, `normalized`, `unnormalized`. Depending on the cluster_type, the inputs to the scatter plot will be different.